net/golioth/settings.c

```diff
@@ -112,6 +111,10 @@ static int finalize_and_send_response(struct golioth_client *client,
 	QCBORError qerr = QCBOREncode_FinishGetSize(&response->encode_ctx, &response_len);
 
 	if (qerr != QCBOR_SUCCESS) {
+		/*
+		 * Note: If the error is 1 (QCBOR_ERR_BUFFER_TOO_SMALL), then try increasing
```

Comment:
We don't have any influence on the numbering of error codes in QCBOR, so I'd rather drop the reference for `1` and just leave `QCBOR_ERR_BUFFER_TOO_SMALL`.

One thing to consider here is to log a specific error message on `QCBOR_ERR_BUFFER_TOO_SMALL` error (with another `if` or with `switch`), so that instead of generic `QCBOREncode_FinishGetSize error: 1 (QCBOR_ERR_BUFFER_TOO_SMALL)` there will be something like `Settings response too large (consider increasing GOLIOTH_SETTINGS_MAX_RESPONSE_LEN)`.